src/genie/libs/parser/iosxe/show_ap.py

```diff
@@ -295,3 +295,782 @@ def cli(self, output=None):
                 }
 
         return ret_dict
+
+
+# ===============================
+# Schema for:
+#  * 'show ap dot11 5ghz channel'
+# ===============================
+class ShowApDot115GhzChannelSchema(MetaParser):
+    """Schema for show ap dot11 5ghz channel."""
+
+    schema = {
+        "channel_assignment": {
+            "chan_assn_mode": str,
+            "chan_upd_int": int,
+            "anchor_time_hour": int,
+            "channel_noise": "Enable",
+            "channel_interference": str,
+            "channel_load": str,
+            "device_aware": str,
+            "clean_air": str,
+            "wlc_leader_name": str,
+            "wlc_leader_ip": str,
+            "last_run_seconds": int,
+            "dca_level": str,
+            "dca_db": int,
+            "chan_width": str,
+            "max_chan_width": int,
+            "dca_min_energy_dbm": float,
+            "chan_energy_min_dbm": float,
+            "chan_energy_average_dbm": float,
+            "chan_energy_max_dbm": float,
+            "chan_dwell_minimum": str,
```

Comment:
why don't we break it as follows (instead of compiling it as a long string as you do when matching the pattern);
```
"chan_dwell_minimum": {
            "hours": int,
            "minutes": int,
            "seconds": float,
}
```

Or maybe just parse as it is a string without doing string manipulation in line 619?